samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -0,0 +1,217 @@
+﻿// <copyright file="AnalyzeConversationsProvider.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscription.Language
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+
+    using Azure;
+    using Azure.AI.Language.Conversations;
+    using Azure.Core;
+    using Connector;
+    using Connector.Serializable.Language.Conversations;
+    using FetchTranscriptionFunction;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Analyze Conversations async client.
+    /// </summary>
+    public class AnalyzeConversationsProvider
+    {
+        private static readonly TimeSpan RequestTimeout = TimeSpan.FromMinutes(3);
+
+        private readonly ConversationAnalysisClient ConversationAnalysisClient;
+
+        private readonly string Locale;
+
+        private readonly ILogger Log;
+
+        public AnalyzeConversationsProvider(string locale, string subscriptionKey, string region, ILogger log)
+        {
+            ConversationAnalysisClient = new ConversationAnalysisClient(new Uri($"https://{region}.api.cognitive.microsoft.com"), new AzureKeyCredential(subscriptionKey));
+
+            Locale = locale;
+            Log = log;
+        }
+
+        public bool IsConversationalPiiEnabled()
+        {
+            return Locale.Contains("en", StringComparison.OrdinalIgnoreCase) && FetchTranscriptionEnvironmentVariables.ConversationPiiSetting != Connector.Enums.ConversationPiiSetting.None;
+        }
+
+        /// <summary>
+        /// API to submit an analyzeConversations async Request.
+        /// </summary>
+        /// <param name="speechTranscript">Instance of the speech transcript.</param>
+        /// <returns>An enumerable of the jobs IDs and errors if any</returns>
+        public async Task<(IEnumerable<string> jobIds, IEnumerable<string> errors)> SubmitAnalyzeConversationsRequestAsync(SpeechTranscript speechTranscript)
+        {
+            speechTranscript = speechTranscript ?? throw new ArgumentNullException(nameof(speechTranscript));
+
+            var data = new AnalyzeConversationsRequest
+            {
+                DisplayName = "IngestionClient",
+                AnalysisInput = new AnalysisInput(new[]
+                {
+                    new Conversation
+                    {
+                        Id = Guid.NewGuid().ToString(),
+                        Language = Locale,
+                        Modality = Modality.transcript,
+                        ConversationItems = speechTranscript.RecognizedPhrases.Where(r => !string.IsNullOrEmpty(r.NBest.First().Lexical))
```

Comment:
Yes I plan to add the chunking as part of the next iteration of the PR. we are also working in parallel to support longer documents in TA

Reply:
I don't know if you'll run into this issue or not. To pass the automated checking when submitting a sample to this repo, I had to have my .csproj target .NET Core 3.1 instead of .NET 6. As a result, the Enumerable.Chunk function wasn't available to me. However, you can get the source for Chunk here:
https://github.com/dotnet/runtime/blob/main/src/libraries/System.Linq/src/System/Linq/Chunk.cs

I had to make a couple of tweaks to get it to work with C# 8, so my revised version is here:
https://github.com/Azure-Samples/cognitive-services-speech-sdk/blob/v-jaswel-patch-3/scenarios/csharp/dotnetcore/call_center/call_center/Chunk.cs

Let me know if I can answer any questions.

Reply:
Currently not planning to handle chunking as TA async Conversation API will start supporting longer conversations.